package.json

```diff
@@ -9,8 +9,10 @@
   "scripts": {
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
     "lint": "tslint -p tsconfig.json",
+    "lint:fix": "tslint -p tsconfig.json --fix",
     "build": "tsc",
-    "test": "jest --config jestconfig.json",
+    "test": "jest --config jestconfig.json --coverage",
+    "test:file": "jest --config jestconfig.json --",
```

Comment:
this trailing -- is correct?

Reply:
Yes. I am using this to run a single test file from the terminal:
```
 npm run test:file src/__tests__/index.test.ts
```